### PR TITLE
Massive update

### PR DESCRIPTION
**Pull request by oterral**
[Test](https://mf-geoadmin3.dev.bgdi.ch/teo_up/?lang=fr&topic=ech&bgLayer=ch.swisstopo.pixelkarte-farbe&X=190000.00&Y=660000.00&zoom=1&dev3d=true&debug)

### CODE REVIEW

**Comment by oterral** [comment]
Ready to review/test

**Comment by oterral** [comment]
Tested on ie 9 (query, draw, exportKML) all works fine.

**Comment by loicgasser** [comment]
this is a good improvement, I'll review tomorrow. Jenkins not happy though.

File: Makefile [hunk 1]
```diff
@@ -256,20 +256,21 @@ prd/robots.txt: scripts/robots.mako-dot-txt .build-artefacts/last-deploy-target
...
 	cp -rf  $^ $@
 
-prd/lib/build.js: src/lib/jquery-2.0.3.min.js \
-		    src/lib/bootstrap-3.3.1.min.js \
+prd/lib/build.js: src/lib/jquery.min.js \
+		    src/lib/bootstrap.min.js \
 		    src/lib/moment-with-customlocales.min.js \
-		    src/lib/typeahead-0.9.3.min.js src/lib/angular.min.js \
+		    src/lib/typeahead-0.9.3.min.js \
+				src/lib/angular.min.js \
```
Comment (loicgasser) [inline Makefile]: indent

**Comment by loicgasser** [comment]
src/lib/localforage.js is commited as exectuable, normal?

**Comment by loicgasser** [comment]
Build simplified I like it, and it works well.

**Comment by loicgasser** [comment]
jenkins test

**Comment by oterral** [comment]
@loicgasser where do you see that localforage is an executable ?

**Comment by loicgasser** [comment]
See https://github.com/geoadmin/mf-geoadmin3/blob/teo_up/src/lib/localforage.js

At top:
Executable File  2471 lines (2088 sloc)  80.3 KB

**Comment by oterral** [comment]
@loicgasser fixed.

**Comment by loicgasser** [comment]
LGTM for me. no backward compatibility issue?

We don't create a fresh checkout of the project when deploying to dev and creating a snpashot.
See https://github.com/geoadmin/mf-geoadmin3/blob/master/scripts/deploydev.sh#L31

Only make cleanall. I think we should change that. I think it should be fine but did not check.

**Comment by oterral** [comment]
I think clean all is enough `cleanall` that puts the folder in the same state as a fresh checkout

**Comment by loicgasser** [comment]
not if you have generated files or folders that are ignored and not cleaned anymore between 2 versions.
but it doesn't seem to be the case here.

**Comment by loicgasser** [comment]
Any body wants to have a look otherwise, I think we're good to merge.

**Pull request merged**